d011-blackjack/Untitled-1.py

```diff
@@ -0,0 +1,128 @@
+#initially, randomly distribute two cards each to house and player 
```

Comment:
Add a little documentation:

- Name of the project/game
- What is the purpose?
- What did you learn from it?

Reply:
- Blackjack game, playing cards between the House and the Player. 
- Random function was used in the game
- A list is capable for both integers and string values

Reply:
I added the same (almost exactly) the the file